test/e2e/provisioning/helper.go

```diff
@@ -33,9 +33,10 @@ var (
 	scenarios = buildScenarios()
 
 	versions = []*semver.Version{
-		semver.MustParse("v1.17.13"),
-		semver.MustParse("v1.18.10"),
-		semver.MustParse("v1.19.3"),
+		semver.MustParse("v1.17.16"),
```

Comment:
In the test we are using K8S version 1.20, using kubelet 1.17 is out of skew policy. It should work anyway, but it's not guaranteed. We should invest some time to address this point (it can be a followup PR if is seems to work).

Reply:
Good point, we should fix this in the near future when we remove 1.17 support.